pkg/pillar/types/assignableadapters.go

```diff
@@ -95,7 +96,13 @@ func (ioType IoType) IsNet() bool {
 // LookupIoBundle returns nil if not found
 func (aa *AssignableAdapters) LookupIoBundle(ioType IoType, name string) *IoBundle {
 	for i, b := range aa.IoBundleList {
-		if b.Type == ioType && strings.EqualFold(b.Name, name) {
+		// XXX the new enums are sent as zero by the controller
```

Comment:
I must be missing something, why are they set as 0 @nordmark ?

Reply:
Because code in the zededa controller which uses the wrong enum with only 4 values for ioType, and that code goes via a string representation so 4 for "audio" becomes 0.

Reply:
This is why audio assignment isn't currently working.